acsoo/templates/project/+project.name+/.pre-commit-config.yaml

```diff
@@ -50,26 +50,24 @@ repos:
       - id: mixed-line-ending
         args: ["--fix=lf"]
   - repo: https://github.com/pycqa/flake8
-    rev: 3.9.2
+    rev: 5.0.4
     hooks:
       - id: flake8
-        name: flake8
-        additional_dependencies: ["flake8-bugbear==21.4.3"]
-  - repo: https://github.com/pre-commit/mirrors-pylint
-    rev: v2.5.3
+        additional_dependencies: ["flake8-bugbear==22.9.11"]
+  - repo: https://github.com/OCA/pylint-odoo
+    rev: 7.0.0
     hooks:
-      - id: pylint
-        name: pylint
+      - id: pylint_odoo
         files: odoo/addons/
         args: []
         additional_dependencies: [pylint-odoo==3.7.1]
```

Comment:
@sbidoul this additional dependency is no more required. Moreover it introduces a version mismatch .

Reply:
Yes, good catch.